### DISCUSSION TITLE
Next CP2K Dev Meeting

### DISCUSSION THREAD

**Discussion by robertschade** [cp2k/cp2k#3262] [General]
Dear @cp2k/cp2k-core-developers @cp2k/cp2k-developers,
we would like to schedule our next CP2K Dev meeting (see also https://www.cp2k.org/dev:meetings) soon. We would be happy if you could fill in your availability in the survey below by the end of next week (16th of February).

https://terminplaner6.dfn.de/p/07da50f50ddf45261b187e4f12e8abbf-593635

We would also like to invite everyone to contribute topics to the meeting. You can either comment here, let me know, or simply let us know during the meeting. If you would like to show some slides, you can either add your content to
https://docs.google.com/presentation/d/1ZblD6WNEJEV4EsfJOmpvMvC_Nxzh0-62ePolLYuR3n4/edit?usp=sharing
(please let me know if you need access)
or show your slides during the meeting with screen sharing.
Many Thanks
Robert

**Comment by robertschade** [comment]
Dear @cp2k/cp2k-core-developers @cp2k/cp2k-developers,
we would like to invite you to our next CP2K dev meeting  (see also https://www.cp2k.org/dev:meetings) on Friday, March 8 th, 14:00-16:00 CET.
We would also like to invite everyone to contribute topics to the meeting. You can either comment here, let me know, or simply let us know during the meeting. If you would like to show some slides, you can either add your content to
https://docs.google.com/presentation/d/1ZblD6WNEJEV4EsfJOmpvMvC_Nxzh0-62ePolLYuR3n4/edit?usp=sharing
(please let me know if you need access)
or show your slides during the meeting with screen sharing.

Zoom Meeting:
https://uni-paderborn-de.zoom-x.de/j/62714284750?pwd=K0IvdnJ3dC9YMy9yY1EvUnkrSTJOQT09
Meeting-ID: 627 1428 4750
Kenncode: 765348

Best Wishes
Robert

**Comment by mattatlincoln** [comment]
Hi Robert,
I don't have edit access to the slides, but upcoming event:

UK computer centre ARCHER2 is running an online workshop on use of CP2K on the 8th April.
If anyone would like to contribute a short presentation (20-30 mins) ideally linked to a tutorial or example page it would be really appreciated. RPA, LS, excited state contributions would be particularly great.

Matt



Prof. Matt Watkins | Professor of Chemical Physics
School of Mathematics and Physics
University of Lincoln. Brayford Pool, Lincoln, Lincolnshire. LN6 7TS
tel: +44 (0)1522 835867
mattatlincoln.github.io<https://mattatlincoln.github.io/>



                       
                        
Situated in the heart of a historic city, the University of Lincoln is committed to transforming lives and communities through our teaching and research. We are listed in the world’s top 150 universities in the Times Higher Education’s (THE) Young University Rankings 2023 and hold a five-star score overall in the QS Stars rating system of global universities. We are one of a select group of universities to achieve the top Gold rating overall and in both aspect ratings for student experience and student outcomes in the national Teaching Excellence Framework (TEF) 2023. We were awarded the Queen's Anniversary Prize for Further and Higher Education in 2023.
                         
The information in this e-mail and any attachments may be confidential. If you have received this email in error please notify the sender immediately and remove it from your system. Do not disclose the contents to another person or take copies.
​
Email is not secure and may contain viruses. The University of Lincoln makes every effort to ensure email is sent without viruses, but cannot guarantee this and recommends recipients take appropriate precautions.
​
The University may monitor email traffic data and content in accordance with its policies and English law. Further information can be found at: https://www.lincoln.ac.uk/legal. 
________________________________

**Comment by robertschade** [comment]
The slides of the meeting can now be found at https://www.cp2k.org/dev:meetings.
